SensioLabs/Security/Crawler/DefaultCrawler.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the SensioLabs Security Checker.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SensioLabs\Security\Crawler;
+
+class DefaultCrawler extends BaseCrawler
```

Comment:
there is no point extending the BaseCrawler as you are not using any logic from it. Just implement the interface directly